src/intTest/resources/success_uat_data/NHS004/screening_full_acceptance_with_nhsack.edifact.dat

```diff
@@ -0,0 +1,43 @@
+UNA:+.? '
+UNB+UNOC:3+000000004400001+000000024600002+091113:1519+34++SCRN_BCS_MEDRPT_V4++1'
+UNH+1+MEDRPT:0:1:RT:NHS004'
+BGM+LSR'
+DTM+137:200911131519:203'
+S01+01'
+NAD+PO+++BCSP Hub Manager'
+RFF+AHI:UNKNOWN'
+SPR+PRO'
+S01+01'
+NAD+MR+++Bowel Cancer Screening Administrato:r'
+RFF+AHI:UNKNOWN'
+SPR+PRO'
+S01+01'
+NAD+MR+M87613:901'
+SPR+ORG'
+S01+01'
+NAD+SLA+++UNKNOWN'
+SPR+DPT'
+S01+01'
+NAD+SLA+++Southern Bowel Cancer Screening Pro:gramme Hub'
+SPR+ORG'
+S02+02'
+GIS+N'
+RFF+SRI:34/1'
+STS++UN'
+DTM+ISR:200911101145:203'
+S06+06'
+ADR++US:HIGHWOOD BUNGALOW:EXETER:DEVON++TQ14 4QJ'
```

Comment:
Do you know if this ADR is correct for a screening message? According to the EDIFACT spec for ADR (which should be the same for both pathology and screening messages), I don't think this is correct, as there should be placeholder colons for 5 address lines.

I think this should be formatted like `ADR++US:HIGHWOOD BUNGALOW:::EXETER:DEVON++TQ14 4QJ'` (note the two extra colons, which are left blank)

Reply:
This is from the example file we were given from Screening.

My interpretation of the EDIFACT spec is that since only the first address line is mandatory, and the rest are optional, we should support EDIFACT messages that do not provide placeholder lines, as we can still extract the data just fine.

I'm happy to be convinced otherwise however!

Reply:
In Teams, @bartek-sarul said trailing colon separators can be omitted

> [Monday 15:44] Bartosz Sarul
>     Andrew Walker Andrew Graham
> if last fields or subfields are empty than the trailing + or : can be omitted 
>  
> ​[Monday 15:45] Bartosz Sarul
>     so 
> QWE+ASD:::++++
> is the same as
> QWE+ASD

Reply:
Yeah that makes sense, but I don't think I've ever seen an example of an EDIFACT segment which omits those separators, all our EDIFACT examples just include the separators (even if there's no data).
But if we're saying we have to support possibly omitted separators in EDIFACT, then yeah we can have a separate story to handle that for `ADR`

Reply:
Note: by "does not support it", the effect is that the address lines will be ignored and only the postcode retained. You should see in the fhir output that the address only has `TQ14 4QJ`, and nothing about Highwood Bungalow, Exeter, Devon.

Reply:
Won't this just result in data being mapped in different place in FHIR?

Reply:
Agreed that it's ok to have empty lines on FHIR side

Reply:
I've updated the test files to match the changes to the address lines.